docs/api/api.md

```diff
@@ -0,0 +1,359 @@
+<!--
```

Comment:
This file is generated automatically or maintain by ourseleves?

Reply:
generated automatically